nuget/lib/dependabot/nuget/credential_helpers.rb

```diff
@@ -0,0 +1,19 @@
+# typed: true
+# frozen_string_literal: true
+
+module Dependabot
+  module Nuget
+    module CredentialHelpers
+      def self.get_token_from_credentials(credentials)
```

Comment:
nit: use singular `credential` in the method name and as the parameter.

Reply:
Done.